js/navigation/navigator.js

```diff
@@ -133,6 +133,7 @@ define(["jquery",
 			if ( isPopStateEvent ) {
 				popstateEvent = new $.Event( "popstate" );
 				popstateEvent.originalEvent = {
+					preventDefault: $.noop,
```

Comment:
If you're mocking an event it's not enough to use a plain object with just one or two properties. It should either be a `jQuery.Event` or a native `event` object. I've done a minimum fix here of adding `preventDefault` so it won't die. I noticed a similar [broken assumption in pagecontainer.js](https://github.com/jquery/jquery-mobile/blob/d04308f591d4d0e58443d6a645d0cc1f599888d2/js/widgets/pagecontainer.js#L131) that might fail because of this but it's not related to jQuery core changes and it's not hit, at least in the unit tests.